### ISSUE TITLE
Takes 2 Clicks to Show Popup Menu

### ISSUE THREAD

**Issue by smartinec** [modernben/iptracker#9]
Initial click in menu bar only flashes the popup menu. It does display on the second click.

**Steps to reproduce the behavior:**
1. Click on the IP displayed in the menu bar
2. Popup menu displays for a split second and then hides
3. Click on the IP displayed in the menu bar a second time
4. Popup menu displays correctly

**Desktop:**
 - OS: MacOS
 - Version Ventura 13.5
 - CPU: Intel

**Additional context**
Same issue happens when clicking on different menu bar item and then clicking on the IP menu bar item. It appears that it needs some kind of focus before it will stay on screen

**Comment by modernben** [comment]
Yep, I am seeing the same thing on my side @smartinec. I'm not sure if this is a platform issue or if it's something that I did. The MenuBar registration is pretty straightforward. I'll see if the discord channel has any insights.
